aten/src/ATen/native/Pool.h

```diff
@@ -323,6 +323,8 @@ max_pool3d_backward_shape_check(
   check_dim_size(indices, ndim, ndim-3, otime);
   check_dim_size(indices, ndim, ndim-2, oheight);
   check_dim_size(indices, ndim, ndim-1, owidth);
+
+  TORCH_CHECK(indices.size(0) > 0, "Expected non-empty indices tensor.");
```

Comment:
```suggestion
  TORCH_CHECK_VALUE(indices.size(0) > 0, "Expected non-empty indices tensor.");
```
To throw a ValueError